### ISSUE TITLE
CSIP5: I don't agree with MoSCoW classification

### ISSUE THREAD

**Issue by hsilva-keep** [DILCISBoard/E-ARK-CSIP#121]
https://dilcisboard.github.io/E-ARK-CSIP/implementation/#41-folder-structure-of-the-cs-ip
`CSIP5: The Information Package folder MUST include a folder named metadata, which MUST include at least all metadata relevant for the whole package`

Following are just two scenarios where this "requirement" does not apply:
- When updating an AIP, using an update SIP, and the update only contains a new representation (ROOT metadata will be empty just to comply)
- When generating an DIP, with one representation with files and metadata (ROOT metadata will be empty just to comply)

Bottom line, I think this "requirement" first **MUST** should be a **CAN**.

**Comment by karinbredenberg** [comment]
Empty folders are showing the structure. Closing the issue with no action.
